mne_bids/utils.py

```diff
@@ -272,18 +272,41 @@ def _infer_eeg_placement_scheme(raw):
     return placement_scheme
 
 
-def _extract_landmarks(dig):
-    """Extract NAS, LPA, and RPA from raw.info['dig']."""
+def _extract_landmarks(dig, assert_coord_frame=FIFF.FIFFV_COORD_HEAD):
+    """Extract NAS, LPA, and RPA from raw.info['dig'].
+
+    Parameters
+    ----------
+    assert_coord_frame : None | int
+        A coordinate frame, as defined in FIFF.FIFFV_COORD_*. If the landmark
+        coordinates are not in this coordinate frame, an exception will be
+        raised. If ``None``, no such checks will be performed.
```

Comment:
The idea here is to prevent silent bugs where we accidentally use the wrong coordinate system.